.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Do NOT delete this template. Its great that you decided to contribute 
+to our project. :) Kindly fill the template accordingly! -->
+### Fixes issue # <number>
```

Comment:
again if you write `fixes #num` github automatically closes the issue and this doesn't happen with `Fixes issue #num`

Reply:
Alright. Fixing it.